src/mount/mount-tool.c

```diff
@@ -99,6 +100,7 @@ static void help(void) {
                "                                  Set automount unit property\n"
                "     --bind-device                Bind automount unit to device\n"
                "     --list                       List mountable block devices\n"
+               "  -u --umount                     Unmount a transient mount point\n"
```

Comment:
Is it still only "transient" mount points that can be unmounted with this command?